tests/functional_tests/transfer.py

```
@@ -61,18 +77,20 @@ def reset(self):
-        for i in range(len(seeds)):
+        self.wallet = [None] * len(self.seeds)
+        for i in range(len(self.seeds)):
             self.wallet[i] = Wallet(idx = i)
             # close the wallet if any, will throw if none is loaded
-            try: self.wallet[i].close_wallet()
-            except: pass
-            res = self.wallet[i].restore_deterministic_wallet(seed = seeds[i])
+            try:
+                self.wallet[i].close_wallet()
+            except Exception:
+                pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)